lib/database.cpp

```diff
@@ -348,7 +348,7 @@ void Database::setOlmSessionLastReceived(const QString& sessionId, const QDateTi
 {
     auto query = prepareQuery(QStringLiteral("UPDATE olm_sessions SET lastReceived=:lastReceived WHERE sessionId=:sessionId;"));
     query.bindValue(":lastReceived", timestamp);
-    query.bindValue(":sessionId", sessionId);
+    query.bindValue(":sessionId", sessionId.toLatin1());
```

Comment:
In this case it's better to restrict the API to `QByteArray`, or even just pass `const QOlmSession&` instead of the id. A bit less encapsulation but the string type of the session id will be correct every time.